packages/syntax-icons/src/Icons.stories.tsx

```diff
@@ -39,7 +39,7 @@ import HeartFilled from "./icons/HeartFilled";
 import HeartUnfilled from "./icons/HeartUnfilled";
 import Help from "./icons/Help";
 import Home from "./icons/Home";
-import IdentifyDocument from "./icons/IdentifyDocument";
+import IdentifyDocument from "./icons/IdentityDocument";
```

Comment:
@christianvuerings  you mentioned before that i should add imports with the script using AST, do we have an example somewhere i can see how to do that?

Reply:
also, can i update the package-json exports the same way?

Reply:
@somethiiing I would suggest doing that in a follow up and using our code mods to do that: https://github.com/Cambly/syntax/tree/main/packages/syntax-codemods/src/transforms

Another approach would be to just write a specific ESLint rule for both files (imports + package.json) which would auto fix them.
For supporting `.json` with ESLint you can use something like https://github.com/ota-meshi/eslint-plugin-jsonc